Fonts/Scripts/download-relative-fonts.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+# This script pulls Skyscanner fonts only if `USE_RELATIVE=1`.
+# It adds them to the `Fonts` pod to make them available to the Example app build.
+
+require 'fileutils'
+
+if ENV["USE_RELATIVE"] == "1"
+  puts "Attempting to pull Skyscanner fonts"
+  FileUtils.mkdir_p 'bpk-fonts-tmp'
+  Dir.chdir("bpk-fonts-tmp") do
+    system "git init"
```

Comment:
Can we surpress the output of these git commands in favour of only outputting our own messaging. Maybe we can hook into `CocoaPods` verbose mode and add output if it's enabled?

Reply:
Yeah we can. All `stdout` is hidden unless you specify `--verbose` anyway